src/lib/string.spec.ts

```diff
@@ -84,4 +84,23 @@ describe("string tests", () => {
   ])("uncapitalize", (value, expected) => {
     expect(uncapitalize(value)).toBe(expected);
   });
+
+  test.each([
+    [null as unknown as string, 10, "", null],
+    [undefined as unknown as string, 10, "", undefined],
+    ["", 10, "", ""],
+    ["hello", 10, "", "hello"],
+    ["hello", 5, "", "hello"],
+    ["hello world", 5, "", "hello"],
+    ["hello world", 8, "", "hello wo"],
+    ["hello world", 11, "", "hello world"],
+    ["hello world", 0, "", ""],
+    ["hello", 3, ">>", "hel>>"],
+    ["test", 2, "", "te"],
+    ["hello world", 5, "...", "hello..."],
+    ["hello world", 8, "...", "hello wo..."],
+    ["hello world", 0, "...", "..."],
+  ])("truncate", (value, maxLength, suffix, expected) => {
+    expect(truncate(value, maxLength, suffix)).toBe(expected);

```

Comment:
add another group of tests that call the truncate function without passing the suffix parameter